### ISSUE TITLE
/stops/nearby not working

### ISSUE THREAD

**Issue by stefanjb-it** [public-transport/hafas-rest-api#17]
I am trying to find the stations near the given coordinates, but I always get the following error:

**Postman:**
```
{
    "error": true,
    "msg": "id must be an IBNR"
}
```
**Console:**
```
{"level":50,"time":1665596577386,"pid":26988,"hostname":"STVL2","err":{"type":"Error","message":"id must be an IBNR",
"stack":"Error: id must be an IBNR\n    
at parseStop (C:\\temp\\hafas_test\\node_modules\\hafas-rest-api\\lib\\parse.js:54:29)\n    
at stop (C:\\temp\\hafas_test\\node_modules\\hafas-rest-api\\routes\\stop.js:36:14)\n    at Layer.handle [as handle_request] 
(C:\\temp\\hafas_test\\node_modules\\express\\lib\\router\\layer.js:95:5)\n    at next 
(C:\\temp\\hafas_test\\node_modules\\express\\lib\\router\\route.js:144:13)\n    at Route.dispatch 
(C:\\temp\\hafas_test\\node_modules\\express\\lib\\router\\route.js:114:3)\n    at Layer.handle [as handle_request] 
(C:\\temp\\hafas_test\\node_modules\\express\\lib\\router\\layer.js:95:5)\n    at 
C:\\temp\\hafas_test\\node_modules\\express\\lib\\router\\index.js:284:15\n    at param 
(C:\\temp\\hafas_test\\node_modules\\express\\lib\\router\\index.js:365:14)\n    at param 
(C:\\temp\\hafas_test\\node_modules\\express\\lib\\router\\index.js:376:14)\n    at Function.process_params 
(C:\\temp\\hafas_test\\node_modules\\express\\lib\\router\\index.js:421:3)"},"msg":"id must be an IBNR"}
```

This is the request I'm using:

`http://localhost:5555/stops/nearby?latitude=47.0687&longitude=15.4061`

I'm using this documentation for the request: https://v5.db.transport.rest/api.html#get-stopsid

The code looks nearly exactly the same as in the example, except that I'm using the oebb profile:
```
const createHafas = require('hafas-client')
const oebbProfile = require('hafas-client/p/oebb')
const createApi = require('hafas-rest-api')

const config = {
	hostname: 'STVL2',
	name: 'station-checker-display',
	homepage: 'https://github.com/someone/my-hafas-rest-api',
	version: '1.0.0',
	aboutPage: false
}
const hafas = createHafas(oebbProfile, 'station-monitor-api')

const api = createApi(hafas, config)

api.listen(5555, (err) => {
	if (err) console.error(err)
})
```

Info:
OS: WIN10 22H1
Node version: 16.17.1

**Comment by stefanjb-it** [comment]
I have tested some requests and looked a bit at the code and found that this:
`http://localhost:5555/locations/nearby/?latitude=47.06727184602459&longitude=15.442097181893473&distance=400`
was working.

Result:
```
[
    {
        "type": "stop",
        "id": "691543",
        "name": "Graz Jakominiplatz (Mitte)",
        "location": {
            "type": "location",
            "id": "691543",
            "latitude": 47.067402,
            "longitude": 15.442146
        },
        "products": {
            "nationalExpress": false,
            "national": false,
            "interregional": false,
            "regional": false,
            "suburban": false,
            "bus": true,
            "ferry": false,
            "subway": false,
            "tram": true,
            "onCall": false
        },
        "distance": 14
    },
    {
        "type": "stop",
        "id": "691646",
        "name": "Graz Jakominiplatz (Jakoministraße)",
        "location": {
            "type": "location",
            "id": "691646",
            "latitude": 47.066692,
            "longitude": 15.44203
        },
        "products": {
            "nationalExpress": false,
            "national": false,
            "interregional": false,
            "regional": false,
            "suburban": false,
            "bus": false,
            "ferry": false,
            "subway": false,
            "tram": true,
            "onCall": false
        },
        "distance": 65
    },
    {
        "type": "stop",
        "id": "691544",
        "name": "Graz Jakominiplatz (Reitschulgasse)",
        "location": {
            "type": "location",
            "id": "691544",
            "latitude": 47.067384,
            "longitude": 15.443171
        },
        "products": {
            "nationalExpress": false,
            "national": false,
            "interregional": false,
            "regional": false,
            "suburban": false,
            "bus": true,
            "ferry": false,
            "subway": false,
            "tram": false,
            "onCall": false
        },
        "distance": 81
    },
    {
        "type": "stop",
        "id": "691011",
        "name": "Graz Jakominiplatz (Gleisdorfer Gasse)",
        "location": {
            "type": "location",
            "id": "691011",
            "latitude": 47.067528,
            "longitude": 15.443405
        },
        "products": {
            "nationalExpress": false,
            "national": false,
            "interregional": false,
            "regional": false,
            "suburban": false,
            "bus": true,
            "ferry": false,
            "subway": false,
            "tram": false,
            "onCall": false
        },
        "distance": 102
    },
    {
        "type": "stop",
        "id": "691040",
        "name": "Graz Opernring",
        "location": {
            "type": "location",
            "id": "691040",
            "latitude": 47.068274,
            "longitude": 15.442569
        },
        "products": {
            "nationalExpress": false,
            "national": false,
            "interregional": false,
            "regional": false,
            "suburban": false,
            "bus": true,
            "ferry": false,
            "subway": false,
            "tram": false,
            "onCall": false
        },
        "distance": 116
    },
    {
        "type": "stop",
        "id": "691025",
        "name": "Graz Jakominiplatz (Radetzkystraße)",
        "location": {
            "type": "location",
            "id": "691025",
            "latitude": 47.066629,
            "longitude": 15.440735
        },
        "products": {
            "nationalExpress": false,
            "national": false,
            "interregional": false,
            "regional": false,
            "suburban": false,
            "bus": true,
            "ferry": false,
            "subway": false,
            "tram": false,
            "onCall": false
        },
        "distance": 126
    },
    {
        "type": "stop",
        "id": "691026",
        "name": "Graz Kaiser-Josef-Platz/Oper",
        "location": {
            "type": "location",
            "id": "691026",
            "latitude": 47.068544,
            "longitude": 15.445895
        },
        "products": {
            "nationalExpress": false,
            "national": false,
            "interregional": false,
            "regional": false,
            "suburban": false,
            "bus": true,
            "ferry": false,
            "subway": false,
            "tram": true,
            "onCall": false
        },
        "distance": 320
    },
    {
        "type": "stop",
        "id": "691053",
        "name": "Graz Dietrichsteinplatz (Schlögelgasse)",
        "location": {
            "type": "location",
            "id": "691053",
            "latitude": 47.066917,
            "longitude": 15.446353
        },
        "products": {
            "nationalExpress": false,
            "national": false,
            "interregional": false,
            "regional": false,
            "suburban": false,
            "bus": true,
            "ferry": false,
            "subway": false,
            "tram": false,
            "onCall": false
        },
        "distance": 324
    }
]
```

The code is still the same as before.

**Comment by derhuerst** [comment]
Which version/branch of `hafas-rest-api` are you using?

**Comment by stefanjb-it** [comment]
I'm using this version from npm: https://www.npmjs.com/package/hafas-rest-api.

**Comment by derhuerst** [comment]
This is a bug in the documentation-generating code. Thank you for reporting it.

It should actually be `/locations/nearby`. You can obtain only stops (instead of POIs as well) by using `/locations/nearby?poi=false&latitude=…&longitude=…`.

**Comment by stefanjb-it** [comment]
Ok, thank you for the quick response. I will close the issue.

**Comment by derhuerst** [comment]
Fix published as `hafas-rest-api@4.1.1`. 🎉
